tasks/get-clair-db.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-clair-db
+spec:
+  steps:
+    - name: get-db
+      image: quay.io/redhat-appstudio/clair-in-ci:latest
```

Comment:
is the image based on image from personal account ? (based on jira story) https://github.com/quay/clair-action/blob/main/Dockerfile#L1

Reply:
Yep it is, I wanted to have the image under control. Meaning if we use one from crozzy, we don't own the state of it.
Do you think that using it directly is good idea?

Reply:
I think you are suggesting that image should be based on ubi?

Reply:
Raised the question of author of **clair-in-CI** if the image could be based on ubi.

Reply:
Quoting Joseph Crosland:

So the image is based on the golang image that is the same for all our clair products: https://quay.io/repository/projectquay/golang?tab=tags it is build on ubi

Reply:
This is not blocking this PR, I wanted only highlight this as risk/issue which could be tracked for future.